src/BUILD_WINDOWS.md

```diff
@@ -1,37 +1,52 @@
-How to build SoftEther VPN for Windows
-======================================
-
-
-Requirements
-------------
-
-You need to install the following software to build SoftEther VPN for Windows.
-
-- Microsoft Windows XP, Vista, 7, 8 or later.
-- Microsoft Visual Studio 2008 with the latest SP (SP1 9.0.30729.4462 QFE).
-  Make sure that you installed the x64 compiler and build tools.
-- Microsoft Windows Driver Kit 7.1.0.
-
-
-Full Build Instructions
------------------------
-
-The following steps will build all SoftEther VPN program files, and also build
-the installer packages of SoftEther VPN. It is very easy.
-
-1. Run the "BuildAll.cmd" batch file in the "src" directory.
-2. Wait until the building process will complete.
-3. The built files are stored on the "output" directory.
-
-
-Partly Build, Debug or Development Instructions on Visual Studio 2008
----------------------------------------------------------------------
-
-If you are a programmer, you can open the SoftEther VPN solution file
-with Visual Studio 2008 to customize. Open "src\SEVPN.sln" and enjoy it.
-
-
-************************************
-Thank You Using SoftEther VPN !
-By SoftEther VPN Open-Source Project
-http://www.softether.org/
+How to build SoftEther VPN for Windows
+======================================
+
+
+Requirements
+------------
+
+You need to install the following software to build SoftEther VPN for Windows.
+
+- Microsoft Windows XP, Vista, 7, 8 or later.
+- Microsoft Visual Studio 2008 with the latest SP (SP1 9.0.30729.4462 QFE).
+  Make sure that you installed the x64 compiler and build tools.
+- Microsoft Windows Driver Kit 7.1.0.
+
+
+Full Build Instructions
+-----------------------
+
+The following steps will build all SoftEther VPN program files, and also build
+the installer packages of SoftEther VPN. It is very easy.
+
+1. Run the "BuildAll.cmd" batch file in the "src" directory.
+2. Wait until the building process will complete.
+3. The built files are stored on the "output" directory.
+
+
+Partly Build, Debug or Development Instructions on Visual Studio 2008
+---------------------------------------------------------------------
+
+If you are a programmer, you can open the SoftEther VPN solution file
+with Visual Studio 2008 to customize. Open "src\SEVPN.sln" and enjoy it.
+
+Visual Studio 2008 is required as to maintain compatibility with Windows 9x, due to Visual C++ 2008 being the last version compatibile with Windows 9x and Windows NT 4.x binary linking.
+
+- Visual Studio 2008's installer ISO can be found on Microsoft's site here: 
+https://download.microsoft.com/download/8/1/d/81d3f35e-fa03-485b-953b-ff952e402520/VS2008ProEdition90dayTrialENUX1435622.iso
```

Comment:
This link is for Visual Studio 2008 Professional, it's a 90 days trial.

Visual Studio 2008 Express: https://go.microsoft.com/fwlink/?LinkId=104679

Reply:
@davidebeatrici I did see that, when I installed it though there was a serial key already added into the install steps with no option to remove it or overwrite it, so I assumed that they stopped activation for it.

Reply:
The serial key is already present because it's not supposed to be changed, as the program is free.

Visual Studio 2008 Express with Service Pack 1: https://download.microsoft.com/download/E/8/E/E8EEB394-7F42-4963-A2D8-29559B738298/VS2008ExpressWithSP1ENUX1504728.iso